api/docs/v2/conf.py

```diff
@@ -106,7 +106,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
```

Comment:
Will fail black formatting.
```suggestion
language = "en"
```

Reply:
`black` would change a _lot_ in the conf file (and some older docs files). Maybe it would be best to keep this PR focused on the H-S content and have a separate chore PR to tidy that up.

Reply:
checked and we don't run lint tests on anything in `api/docs` — on purpose, i presume. we could add the .py files, but the .rst files should be kept out of it for sure, because hard-wrapping prose makes it _harder_ to edit and maintain over time.